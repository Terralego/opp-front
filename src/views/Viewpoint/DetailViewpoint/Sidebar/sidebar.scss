@import '../../../../variables';

.sidebar {
    height: calc( 100vh - 50px );
    width: 410px;
    position: absolute;
    right: 0;
    top: 50px;
    background-color: $sidebarDarkblue;
    box-shadow: 0px 0 4px 0 rgba(0,0,0,0.50);
}
