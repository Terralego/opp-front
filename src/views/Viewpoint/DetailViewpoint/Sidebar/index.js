import React from 'react';
import SidebarTabsNav from '../SidebarTabsNav';
import SidebarBottomActions from '../SidebarBottomActions';

import './sidebar.scss';

const Sidebar = () => (
  <div className="sidebar">
    <SidebarTabsNav />
    <SidebarBottomActions />
  </div>
);

export default Sidebar;
