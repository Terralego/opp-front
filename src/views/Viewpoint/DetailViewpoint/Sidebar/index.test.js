import React from 'react';
import renderer from 'react-test-renderer';

import Sidebar from '.';

jest.mock('react-router-dom', () => ({
  NavLink: () => null,
  withRouter: ({ children }) => children,
  Route: () => null,
  Switch: () => null,
  Link: () => null,
}));

jest.mock('../SidebarTabsNav', () => () => <p>SidebarTabsNav</p>);
jest.mock('../SidebarBottomActions', () => () => <p>SidebarBottomActions</p>);

it('should render', () => {
  const tree = renderer.create(<Sidebar />).toJSON();
  expect(tree).toMatchSnapshot();
});
