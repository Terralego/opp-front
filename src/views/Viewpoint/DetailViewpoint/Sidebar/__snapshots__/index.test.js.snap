// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="sidebar"
>
  <p>
    SidebarTabsNav
  </p>
  <p>
    SidebarBottomActions
  </p>
</div>
`;
